### PR TITLE
Add K8s Support to Apache Solr

### PR DESCRIPTION
**Pull request by Caleb-Hurshman**
This PR adds support for K8s deployments of Apache Solr.

### CODE REVIEW

File: apache-solr-mixin/config.libsonnet [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   _config+:: {
-    enableMultiCluster: false,
+    enableMultiCluster: true,
```
Comment (JonathanWamsley) [inline apache-solr-mixin/config.libsonnet:3]: I believe this should be false by default still

**Review by JonathanWamsley — COMMENTED** [review]
Overall looks pretty good. I think the logs overview needs a cluster selector too now right?

**Comment by Caleb-Hurshman** [comment]
> Overall looks pretty good. I think the logs overview needs a cluster selector too now right?

I thought so too, but looking at the docs and previous examples, we've never changed the log dashboards. Maybe I'm looking in the wrong places, lmk if you know something I'm forgetting.

**Comment by JonathanWamsley** [comment]
> > Overall looks pretty good. I think the logs overview needs a cluster selector too now right?
> 
> I thought so too, but looking at the docs and previous examples, we've never changed the log dashboards. Maybe I'm looking in the wrong places, lmk if you know something I'm forgetting.

I think they did not have the individual logs dashboard. It seems you updated the selector though for [couchbase](https://github.com/grafana/jsonnet-libs/blob/664ac8fda54706b3b0a089f3b47de40644dade5b/couchbase-mixin/dashboards/couchbase-node-overview.libsonnet#L1019) before. You can add an `cluster` label in the newer logs dashboard like with [clickhouse](https://github.com/grafana/jsonnet-libs/blob/6f568d7bc14260f10321f1344f53cfb2d6fc146f/clickhouse-mixin/dashboards/addlogs.libsonnet#L13) .

**Review by JonathanWamsley — APPROVED** [review]
Looks good 👍

**Review by Dasomeone — APPROVED** [review]
LGTM!

**Pull request merged**